src/DocBlock/Annotation.php

```diff
@@ -157,7 +157,7 @@ public function getTypes()
      */
     public function setTypes(array $types)
     {
-        $pattern = '/'.preg_quote($this->getTypesContent()).'/';
+        $pattern = '/'.preg_quote($this->getTypesContent(), '/').'/';
```

Comment:
reasoning, like new unit test, is missing.
please provide any case when value we quoting here actually contains `/`.